src/drawers/rectDrawer.ts

```diff
@@ -11,8 +11,8 @@ export module Drawers {
     private _measurer: SVGTypewriter.Measurers.CacheCharacterMeasurer;
     private _writer: SVGTypewriter.Writers.Writer;
 
-    constructor(key: string, isVertical: boolean) {
-      super(key);
+    constructor(dataset: Dataset, isVertical: boolean) {
```

Comment:
There is some weirdness in which the text needs to know information about the plot's verticalness to write the text.  I will highly consider removing this constructor argument when I do remove it.